var/spack/repos/builtin/packages/intel-oneapi-runtime/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+import os
+
+from llnl.util import tty
+
+from spack.package import *
+from spack.pkg.builtin.gcc_runtime import get_elf_libraries
+
+
+class IntelOneapiRuntime(Package):
+    """Package for OneAPI compiler runtime libraries"""
+
+    homepage = "https://software.intel.com/content/www/us/en/develop/tools/oneapi.html"
+    has_code = False
+
+    tags = ["runtime"]
+
+    requires("%oneapi")
+
+    depends_on("gcc-runtime", type="link")
+
+    LIBRARIES = [
+        "imf",
+        "intlc",
+        "irng",
+        "svml",
+        "ifcore",  # Fortran
+        "ifcoremt",  # Fortran
+        "ifport",  # Fortran
+        "iomp5",
+        "sycl",
```

Comment:
Not clear to me where to look :worried: 
```console
$ find /opt/intel/oneapi/compiler -name redist.txt 
$ find /opt/intel/oneapi/compiler -name *.txt | wc -l
81
```

Reply:
@rscohn2 Do you have any idea where I need to look?

Reply:
These files are redistributable, but redistribution carries some obligations and this is not a case they anticipated. Redistribution usually means giving to another legal entity. It looks like you are just copying the files to a different directory? If that is all you are doing then I don't think there are license issues.

Reply:
Sure, @rscohn2 The point is though that those other directories can then be packaged in a buildcache and redistributed via a public mirror, for instance from our `oneapi` pipeline. Would that be ok?

Reply:
I think the intention for the redistribution license was that you are
distributing an application that needs the libraries. This looks like
something different. Rather than try to get lawyers involved, could you
mark this package as non cacheable? That is what I did for all the
intel-oneapi-* packages. I recognize that it is less convenient.

https://github.com/spack/spack/blob/develop/lib/spack/spack/build_systems/oneapi.py#L30

Reply:
@rscohn2 

> https://github.com/spack/spack/blob/develop/lib/spack/spack/build_systems/oneapi.py#L30

I was not aware of this class level attribute. Do you know how it is used? From what I can tell:
```console
$ grep redistribute_source -R lib
lib/spack/spack/build_systems/oneapi.py:    redistribute_source = False
```
it is not used directly in Spack?

Reply:
I did not know it was never implemented: https://github.com/spack/spack/pull/20185

I suggest you keep the copying code as it is now. It seems more robust and consistent with gcc implementation. I will look at implementing the feature that prevents packages from being published. Is @scheibelp the right person to ask about it.

Reply:
@rscohn2 Thanks for the reply. I think @scheibelp is the correct person to continue working on #20185